multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/Madara.kt

```diff
@@ -665,14 +665,16 @@ abstract class Madara(
                 .toMutableSet()
 
             // add tag(s) to genre
+            val mangaTitle = if (manga.initialized) manga.title else "not initialized"
```

Comment:
You are doing it the wrong way. `manga.initialized` is a custom field and is always `false` because you just created it.

```suggestion
            val mangaTitle = try {
                manga.title
            } catch (_: UninitializedPropertyAccessException) {
                "not initialized"
            }
```